modules/account/README.md

```diff
@@ -45,13 +47,13 @@ components:
                 - DenyLeavingOrganization
             - name: platform
               accounts:
-                - name: platformprod
+                - name: prodplatform
                   tags:
                     eks: true
-                - name: platformdev
+                - name: devplatform
                   tags:
                     eks: true
-                - name: platformstaging
+                - name: stageplatform
```

Comment:
Why are these modified ?

Reply:
Copypasta — I can restore

Reply:
Tbh this stays more consistent with `proddata` — I think it's better actually